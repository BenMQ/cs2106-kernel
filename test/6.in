cr a 2
cr b 2
cr c 2
cr d 2
req R4 4
to
req R4 1
req R4 2
req R4 1
rel R4 3
to
to
rel R4 1
to
to
to
rel R4 1
de b