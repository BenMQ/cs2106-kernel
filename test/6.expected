init a a a a a b c d a a b c c a b d d error 